pkgs/racket-doc/scribblings/reference/numbers.scrbl

```diff
@@ -6,8 +6,7 @@
                      racket/unsafe/ops
                      racket/require
                      racket/random
-                     racket/list
-                     math/flonum))
+                     racket/list))
```

Comment:
This `require` of `math/flonum` seemed to have no uses, or at least I couldn't find them.

Reply:
Ah, it's for `fllogb`